content/zh/docs/reference/kubernetes-api/authorization-resources/role-binding-v1.md

```diff
@@ -0,0 +1,718 @@
+---
+api_metadata:
+  apiVersion: "rbac.authorization.k8s.io/v1"
+  import: "k8s.io/api/rbac/v1"
+  kind: "RoleBinding"
+content_type: "api_reference"
+description: "RoleBinding 引用一个角色，但不包含它。"
+title: "RoleBinding"
+weight: 8
+auto_generated: false
+---
+<!--
+---
+api_metadata:
+  apiVersion: "rbac.authorization.k8s.io/v1"
+  import: "k8s.io/api/rbac/v1"
+  kind: "RoleBinding"
+content_type: "api_reference"
+description: "RoleBinding references a role, but does not contain it."
+title: "RoleBinding"
+weight: 8
+auto_generated: true
+---
+-->
+`apiVersion: rbac.authorization.k8s.io/v1`
+
+`import "k8s.io/api/rbac/v1"`
+
+## RoleBinding {#RoleBinding}
+<!--
+RoleBinding references a role, but does not contain it.  It can reference a Role in the same namespace or a ClusterRole in the global namespace. It adds who information via Subjects and namespace information by which namespace it exists in.  RoleBindings in a given namespace only have effect in that namespace.
+-->
+RoleBinding 引用一个角色，但不包含它。
+RoleBinding 可以引用相同命名空间中的 Role 或全局命名空间中的 ClusterRole。
+RoleBinding 通过 Subjects 和所在的命名空间信息添加主体信息。
+处于给定命名空间中的 RoleBinding 仅在该命名空间中有效。
+
+<hr>
+
+- **apiVersion**: rbac.authorization.k8s.io/v1
+
+- **kind**: RoleBinding
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata.
+
+- **roleRef** (RoleRef), required
+
+  RoleRef can reference a Role in the current namespace or a ClusterRole in the global namespace. If the RoleRef cannot be resolved, the Authorizer must return an error.
+
+  <a name="RoleRef"></a>
+  *RoleRef contains information that points to the role being used*
+-->
+  标准的对象元数据。
+
+- **roleRef** (RoleRef)，必需
+  
+  roleRef 可以引用当前命名空间中的 Role 或全局命名空间中的 ClusterRole。
+  如果无法解析 roleRef，则 Authorizer 必定返回一个错误。
+  
+  <a name="RoleRef"></a>
+  **roleRef 包含指向正被使用的角色的信息。**
+<!--
+  - **roleRef.apiGroup** (string), required
+    APIGroup is the group for the resource being referenced
+
+  - **roleRef.kind** (string), required
+    Kind is the type of resource being referenced
+
+  - **roleRef.name** (string), required
+    Name is the name of resource being referenced
+-->  
+  - **roleRef.apiGroup** (string)，必需
+    
+    apiGroup 是被引用资源的组
```

Comment:
Is it intentional to change `APIGroup` to `apiGroup`?

Reply:
yes, keep same as shown in **roleRef.apiGroup**. @tengqm suggested several times in #33828 and #33892.

Reply:
APIGroup is the name in the Go struct, so in many source code comments, developers write the word as `APIGroup`. However, when serialized into JSON or YAML (the user-facing format), it is always `apiGroup`. We should document the latter rather than how the implementation looks like.